examples/examples/homogeneous_elastic/forward_2D.py

```diff
@@ -0,0 +1,122 @@
+
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.lines as lines
+
+from stride import *
+from stride.utils import wavelets
+
+
+async def main(runtime):
+    # Create the grid
+    extent = (120e-3, 70e-3)  # [m]
+    spacing = (0.5e-3, 0.5e-3)  # [m]
+
+    shape = tuple([int(each_extent / each_spacing) + 1 for each_extent, each_spacing in zip(extent, spacing)])  # [grid_points]
+    extra = (50, 50)  # boundary [grid_points]
+    absorbing = (40, 40)  # absorbing boundary [grid_points]
+
+    space = Space(shape=shape,
+                  extra=extra,
+                  absorbing=absorbing,
+                  spacing=spacing)
+    print('Space complete')
```

Comment:
This sort of printing should not be in the file

Reply:
Correction made